src/liquid/components/ld-pagination/ld-pagination.shadow.css

```diff
@@ -34,13 +54,44 @@
   margin-right: var(--ld-pagination-items-space);
 }
 
+.ld-pagination__arrow {
+  &--left {
+    ld-button {
+      margin-right: calc(var(--ld-pagination-items-space) * -1);
+    }
+  }
+
+  &--right {
+    ld-button {
+      margin-left: calc(var(--ld-pagination-items-space) * -1);
+    }
+  }
+}
+
```

Comment:
```suggestion
```
I think this may not even be necessary. At least it looks more according to design, if I remove those classes (for small spacing). For normal spacing the difference is (I think) neglectable  and for large spacing (if needed) there is not even a difference.